src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -936,7 +936,7 @@ private static boolean isPrecededByJavadoc(DetailAST token) {
      */
     private static boolean isTypeField(DetailAST variableDef) {
         return TokenUtil.isOfType(variableDef.getParent().getParent(),
```

Comment:
Interfaces can have fields,too (if they should is another discussion). Can we just use TokenUtil#isTypeDeclaration here?

Reply:
yes, you are right. I fixed this

Reply:
Resolved.